runtime/include/tt/runtime/detail/debug.h

```diff
@@ -63,14 +63,17 @@ struct Hooks {
 #endif
   }
 
+  void unregisterHooks() const { operatorCallback = std::nullopt; }
```

Comment:
Oh there's also a build failure:
```
/__w/tt-mlir/tt-mlir/runtime/include/tt/runtime/detail/debug.h:66:34: error: use of undeclared identifier 'operatorCallback'
   66 |   void unregisterHooks() const { operatorCallback = std::nullopt; }
      | 
```

`operatorCallback` is only defined for debug builds.  Probably the right fix is:
```c++
void unregisterHooks() const {
#if defined(TT_RUNTIME_DEBUG) && TT_RUNTIME_DEBUG == 1
  operatorCallback = std::nullopt;
#endif
}
```